HC 967 update trademark articles and categories (#43)
arduino/help-center-content: content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md

## Name

You may include the word Arduino in the title of your course, workshop or publication if you use it in a descriptive way.
To avoid violating Arduino trademarks, you must name your course in a way that does not suggest it is officially endorsed, approved, or certified by Arduino.

<table>
  <tr>
    <th style="text-align: center;">Do</th>
    <th style="text-align: center;">Don't</th>

  </tr>
  <tr>
    <td>Basic tutorial for Arduino</td>
    <td>Arduino basic tutorial</td>
  </tr>
  <tr>
</table>

## Licenses and fees

If you decide to open a website about Arduino, write a book, or develop a course or a workshop, you are free to do so, and you do not have to ask us for permission or pay any license fees. Similarly, if you commercialize your course or publication, you do not have to pay Arduino any royalty fees.

## Course content

You are allowed to use screenshots of Arduino IDE in your course content, provided that you mention its source. You are also free to use images of Arduino products in your course or publication.

> All content on our website, including text and images, is distributed under Creative Commons license. This means you can use them on your own derived works, in part or completely, as long as you also adopt the same license.

Reviewer: ```suggestion
All content on our website, including text and images, is distributed under a Creative Commons license. This means you can use them on your own derived works, in part or completely, as long as you also adopt the same license.
```
Also: which CC license?

Author: See [7a3100a](https://github.com/arduino/help-center-content/pull/43/commits/7a3100aba033099549130334d74237c6c939e73b)

* See [Software and Licensing based on Arduino](https://support.arduino.cc/hc/en-us/articles/4415094490770-Distributing-products-based-on-Arduino).

> If your content includes the use of components (for example, if you develop a video tutorial), you are free to use whichever brand you choose. In other words, you do not have to use original Arduino products. However, you must provide correct descriptions of the components you use. If you are using a compatible board, you may not call it “Arduino”.

> ## Trademark Acknowledgments

Reviewer: ```suggestion
## Trademark acknowledgments
```

> All uses of the trademarks of Arduino SA should include an acknowledgement (for example: “Arduino® is a trademark of Arduino SA”). Please note that you should also include the registered trademark symbol the first time Arduino is mentioned in text: ARDUINO® or Arduino®.

Reviewer: ```suggestion
All uses of the trademarks of Arduino SA should include an acknowledgment (for example: “Arduino<sup>®</sup> is a trademark of Arduino SA”). Please note that you should also include the registered trademark symbol the first time Arduino is mentioned in text: ARDUINO<sup>®</sup> or Arduino<sup>®</sup>.
```
